types/element-resize-detector/index.d.ts

```diff
@@ -1,20 +1,37 @@
-// Type definitions for element-resize-detector 1.1
+// Type definitions for element-resize-detector 1.2
```

Comment:
This should match the real version of the package -- which hasn't published 1.2 yet.

Reply:
ERROR: 1:25  dt-header  Error parsing header. Expected: foo MAJOR.MINOR (patch version not allowed). See: https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md

Reply:
Seems like I can only bump up the minor version

Reply:
Reverted back to 1.2 since 1.1.14 will break the build

Reply:
I think you should just leave it at 1.1; old versions of the package are still available on npm if these changes break others. But that's unlikely since the changes are strictly additive.